### PR TITLE
[SERVMAN] Use the ShellAboutW API routine to display the about dialog

### PR DESCRIPTION
**Pull request by KyleKatarn31**
## Purpose

As the dialog simply displays the copyright names and nothing particular it makes sense to use ShellAboutW() instead.

image

Similar to https://github.com/reactos/reactos/pull/2743 by @HBelusca

## Proposed changes

image

### CODE REVIEW

**Comment by KyleKatarn31** [comment]
@learn-more @Fraizeraust : Same as https://github.com/reactos/reactos/pull/2751, including received comments up to now.

File: base/applications/mscutils/servman/lang/es-ES.rc [hunk 1]
```diff
@@ -334,6 +322,7 @@ END
 STRINGTABLE
 BEGIN
     IDS_APPNAME "Administrador de servicios de ReactOS"
+    IDS_APPAUTHORS "Copyright (C) 2005-2007 by Ged Murphy (gedmurphy@reactos.org)"
```
Comment (HBelusca) [inline base/applications/mscutils/servman/lang/es-ES.rc:325]:
```suggestion
    IDS_APPAUTHORS "Copyright (C) 2005-2007 creado por Ged Murphy (gedmurphy@reactos.org)"
```

File: base/applications/mscutils/servman/mainwnd.c [hunk 2]
```diff
@@ -590,11 +594,12 @@ MainWndCommand(PMAIN_WND_INFO Info,
         break;
 
         case ID_ABOUT:
-            DialogBox(hInstance,
-                      MAKEINTRESOURCE(IDD_ABOUTBOX),
-                      Info->hMainWnd,
-                      AboutDialogProc);
-            SetFocus(Info->hListView);
+            LoadStringW(hInstance, IDS_APPNAME, szAppName, sizeof(szAppName) / sizeof(WCHAR));
```
Comment (HBelusca) [inline base/applications/mscutils/servman/mainwnd.c:597]: Use either `ARRAYSIZE(szAppName)` or `_countof(szAppName)`, whichever is used elsewhere in this app.

File: base/applications/mscutils/servman/mainwnd.c [hunk 3]
```diff
@@ -590,11 +594,12 @@ MainWndCommand(PMAIN_WND_INFO Info,
...
+            LoadStringW(hInstance, IDS_APPAUTHORS, szAppAuthors, sizeof(szAppAuthors) / sizeof(WCHAR));
```
Comment (HBelusca) [inline base/applications/mscutils/servman/mainwnd.c:598]: Ditto.

File: base/applications/mscutils/servman/lang/es-ES.rc (hunk 1 above)
Comment (KyleKatarn31) [reply base/applications/mscutils/servman/lang/es-ES.rc:325]: ok, even if i'd prefer to have translation updated aside from this PR.

File: base/applications/mscutils/servman/mainwnd.c (hunk 2 above)
Comment (KyleKatarn31) [reply base/applications/mscutils/servman/mainwnd.c:597]: ok

File: base/applications/mscutils/servman/mainwnd.c (hunk 3 above)
Comment (KyleKatarn31) [reply base/applications/mscutils/servman/mainwnd.c:598]: ok

**Comment by KyleKatarn31** [comment]
Thanks @HBelusca . Your comments have been applied to https://github.com/reactos/reactos/pull/2754 too

**Comment by KyleKatarn31** [comment]
Thanks

**Pull request merged**